chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2275,6 +2281,7 @@ public void setUseCustomWaterColor(boolean value) {
     if (!actorArray.isEmpty()) {
       json.add("actors", actorArray);
     }
+    json.add("octreeImplementation", octreeImplementation.ordinal());
```

Comment:
A safer (and more human-readable) way to save the octree implementation would be to save the `octreeImplementation.name()` and then load it with `Octree.ImplementationEnum.valueOf(json.get("octreeImplementation")`. This is what's used to save all other enums in the scene config.

Reply:
Now using a string directly